Tests/SwiftMemcacheTests/IntegrationTest/MemcachedIntegrationTests.swift

```diff
@@ -84,4 +85,50 @@ final class MemcachedIntegrationTest: XCTestCase {
             XCTFail("Failed to connect to Memcached server: \(error)")
         }
     }
+
+    func testMemcachedConnectionActor() async throws {
+        let ELG = MultiThreadedEventLoopGroup(numberOfThreads: 1)
+        let connectionActor = MemcachedConnection(host: "memcached", port: 11211, eventLoopGroup: ELG)
+        Task {
+            try await connectionActor.run()
+        }
```

Comment:
Can you please us a task group here `withTaskGroup` and spawn the run method in one child task. `Task {}` is in general to be avoided due to its unstructuredness